app/server-test/src/test/scala/org/bitcoins/server/MockObjects.scala

```diff
@@ -0,0 +1,209 @@
+package org.bitcoins.server
+
+import akka.actor.ActorSystem
+import org.bitcoins.chain.api.ChainApi
+import org.bitcoins.chain.config.ChainAppConfig
+import org.bitcoins.chain.models.{
+  BlockHeaderDb,
+  CompactFilterDb,
+  CompactFilterHeaderDb
+}
+import org.bitcoins.core.bloom.BloomFilter
+import org.bitcoins.core.crypto.{
+  AesPassword,
+  DoubleSha256Digest,
+  DoubleSha256DigestBE,
+  MnemonicCode
+}
+import org.bitcoins.core.currency.{Bitcoins, CurrencyUnit}
+import org.bitcoins.core.gcs.FilterHeader
+import org.bitcoins.core.hd.{AddressType, HDPurpose}
+import org.bitcoins.core.p2p.CompactFilterMessage
+import org.bitcoins.core.protocol.blockchain.BlockHeader
+import org.bitcoins.core.protocol.script.ScriptPubKey
+import org.bitcoins.core.protocol.transaction.{EmptyTransaction, Transaction}
+import org.bitcoins.core.protocol.{BitcoinAddress, BlockStamp}
+import org.bitcoins.core.util.FutureUtil
+import org.bitcoins.core.wallet.fee.FeeUnit
+import org.bitcoins.node.SpvNodeCallbacks
+import org.bitcoins.node.config.NodeAppConfig
+import org.bitcoins.node.models.Peer
+import org.bitcoins.wallet.api._
+import org.bitcoins.wallet.config.WalletAppConfig
+import org.bitcoins.wallet.models.{AccountDb, AddressDb, SpendingInfoDb}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object MockObjects {
```

Comment:
Is there any way we can just integrate this wit hthe `Fixture` system rather than creating these mock objects with a bunch of unimplemented methods?

Reply:
@Christewart I don't think we need an integration test just to verify that the URI and form body parser works as expected. I can replace `???` with something else if it's an issue.